src/utilities/table/table.ts

```diff
@@ -159,3 +160,26 @@ export function stringFiltering(columnData: Array<string>, dataIndexes: number[]
     });
     return filteredDataIndexes;
 }
+
+export function hasFilterFunc(controlHeader: Map<string, ControlHeader>, queryResult: Map<number, ProcessedColumnData>): boolean {
```

Comment:
I prefer we rename it as `hasFilter`.

Reply:
There are hasFilter in other files. It may get confusing if we use the same function name.

Reply:
`getHasFilter` looks good to me. 👍